### ISSUE TITLE
Suggestions with instructions and running the program

### ISSUE THREAD

**Issue by LekaiSong** [3DMBDP/3D_Drone_Reconstruction#2]
Good work！
The instructions and Readme are so clear that I can get test results easily.
However, there is a minor problem I met when running your project. It seems like test_SURF.py can only be run with lower version 'OpenCV-contrib-python'. '3.3.0.10' version works.
Also, the link of the demo is invalid.
Looking forward to your updates.

**Comment by Ayush-Shirsat** [comment]
Yeah you are right. SIFT and SURF are not available in OpenCV as they are called “non-free” algorithms. Both SIFT and SURF are patented algorithms, meaning they require permission to use them in commercial algorithms (they are free to use for academic and research purposes though).

Because of this, OpenCV has made the decision to move patented algorithms (along with experimental implementations) to the opencv_contrib package. This means that to obtain access to SIFT and SURF, you’ll need to compile and install OpenCV 3 from source with opencv_contrib  support enabled. 

You can read more in the following link:
Link: https://www.pyimagesearch.com/2015/07/16/where-did-sift-and-surf-go-in-opencv-3/
